services/ui_backend_service/api/ws.py

```diff
@@ -108,6 +108,8 @@ async def subscribe_to(self, ws, uuid: str, resource: str, since: int):
         await self.unsubscribe_from(ws, uuid)
 
         # Create new subscription
+        # TODO: this is only for testing purposes, a quick and dirty cleanup of invalid query params!
+        resource = resource.replace("&postprocess=false", "")
```

Comment:
Do we keep this? I assume `?postprocess=` will not work

Reply:
removed for now, UI takes care of passing proper params. the subscription filter params handling needs a more thorough rework in another PR